easybuild/easyblocks/g/gcc.py

```diff
@@ -525,7 +525,7 @@ def sanity_check_step(self):
             bin_files.append('gfortran')
             lib_files.extend(['libgfortran.%s' % sharedlib_ext, 'libgfortran.a'])
 
-        if 'lto' in self.cfg['languages']:
+        if self.cfg['withlto']:
```

Comment:
since listed `lto` in `languages` is another way of enabling LTO support, shouldn't this be:

```python
if self.cfg['withlto'] or 'lto' in self.cfg['languages']:
```